nodes/nodes/nodes-nodes-managing.adoc

```diff
@@ -21,3 +21,18 @@ configuration of nodes. By creating an instance of a `KubeletConfig` object, a m
 // assemblies.
 
 include::modules/nodes-nodes-managing-about.adoc[leveloffset=+1]
+include::modules/nodes-nodes-working-master-schedulable.adoc[leveloffset=+1]
+include::modules/nodes-nodes-working-setting-booleans.adoc[leveloffset=+1]
+include::modules/nodes-nodes-kernel-arguments.adoc[leveloffset=+1]
+ifdef::openshift-webscale[]
+include::modules/nodes-nodes-rtkernel-arguments.adoc[leveloffset=+1]
+endif::openshift-webscale[]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For information about enabling cgroup v2 during installation, see the xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installation-configuration-parameters-optional_installing-bare-metal[Optional parameters table] in the *Installation configuration parameters* section of your installation process.
```

Comment:
If this specific to BM?
If yes, since we are already pointing to the table, do we need the additional info `in the *Installation configuration parameters* section of your installation process.`?

Reply:
@snarayan-redhat Thank you for pointing this out. The process for installing cgroups at install changed in 4.12. The parameter has been removed. I think we should remove this additional reference.